src/Sylius/Bundle/ReviewBundle/Validator/Constraints/UniqueUserEmail.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ReviewBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @author Mateusz Zalewski <mateusz.p.zalewski@gmail.com>
+ */
+class UniqueUserEmail extends Constraint
+{
+    /**
+     * @var string
+     */
+    public $message = 'sylius.review.author.already_exists';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function validatedBy()
+    {
+        return 'unique_user_email_validator';
```

Comment:
We need this validation for case, when someone is not logged in and want to add a new review with an e-mail, which is related to existing user account (not customer).